src/routes/landing/components/CitySelector.js

```diff
@@ -76,10 +122,45 @@ class CitySelector extends React.PureComponent<PropsType> {
     }, [])
   }
 
+  renderListByLocation (): React.Node {
+    if (this.props.currentLatitude === null || this.props.currentLongitude === null) {
+      return null
+    }
+    const currentLatitude = this.props.currentLatitude
+    const currentLongitude = this.props.currentLongitude
+    let cities = this.props.cities.filter(_city => _city.live)
+      .sort((a: CityModel, b: CityModel) => compareDistance(a, b, currentLongitude, currentLatitude))
+    const numberOfClosestCities = 3
+    cities = cities.slice(0, numberOfClosestCities)
+    const maximalDistance = 90
+    cities = cities.filter(_city => currentDistance(_city, currentLongitude, currentLatitude) < maximalDistance)
+    if (cities && cities.length) {
+      return <View>
+        <CityGroup theme={this.props.theme}> Close </CityGroup>
+        {cities.map(city => <CityEntry
+          key={city.code}
+          city={city}
+          filterText={this.props.filterText}
+          navigateToDashboard={this.props.navigateToDashboard}
+          theme={this.props.theme} />)}
+      </View>
+    } else {
+      alert('There are no cities offering integreat close to you.')
```

Comment:
We shouldn't use `alert` in the render method.
Instead return `<CityGroup theme={this.props.theme}>No Integreat places nearby</CityGroup>` or similar.